src/common/Screen.js

```diff
@@ -44,7 +51,14 @@ class Screen extends React.Component {
           barStyle="dark-content"
           hidden={this.props.orientation === 'LANDSCAPE'}
         />
-        <ModalNavBar title={title} popRoute={popRoute} nav={nav} />
+        <View>
```

Comment:
I guess I wanted to do something with it but it turned out that its not needed and I forgot to remove it -  thank you for pointing this out, for newbies its much more harder to notice such things.